mcp_run_python/main.py

```diff
@@ -181,32 +193,35 @@ def _deno_run_args(
     mode: Mode,
     *,
     http_port: int | None = None,
+    http_host: str | None = None,
     dependencies: list[str] | None = None,
     return_mode: Literal['json', 'xml'] = 'xml',
     allow_networking: bool = True,
 ) -> list[str]:
-    args = ['run']
+    args = ["run"]
```

Comment:
All of these edits should really not be necessary, can you please make the diff as small as possible?